### ISSUE TITLE
Qwen Lora ARA didn't change style - medieval art

### ISSUE THREAD

**Issue by Holmes2002** [ostris/ai-toolkit#376]
<img width="1684" height="686" alt="Image" src="https://github.com/user-attachments/assets/bc01bc4b-7274-48e9-961c-32c9ef0e0965" />

Hi, I finetuned Lora 32 rank with ARA 3bit quantization training with 80samples about medieval art but generation didn't work. Any advices for me pls?

**Comment by UIMicro** [comment]
1. Dont forget to add trigger world to test prompt
2. Try use a significant larnger learning rate e.g. 0.01 and you will see the style change soon, which means the lora is working. But this will introduce large instability to training so I suggest keeping the default or multiply it by 2. It usually takes 5000-6000 steps to see significant change.
3. Rank 32 is a lot for Qwen-Image, since the base model itself is large enough.

**Comment by Holmes2002** [comment]
@UIMicro Hi, I changed my lr from 0.001 to 0.0001 but it caused two case:
1. Model did fit in style prompts, other prompts model still generated normal image in small lr 0.0001
2. Model did underfit in style prompts and original ones in large lr

I didn't know how extractly finetune Lora model and where was problem ?
